apps/governance/src/app/context/StakedToken.tsx

```diff
@@ -25,8 +23,14 @@ const ContractProvider: FC = ({ children }) => {
 }
 
 export const StakedTokenProvider: FC = ({ children }) => {
+  const clients = useApolloClients()
+  const stakingQuery = useStakingQuery({
+    client: clients.staking,
+    fetchPolicy: 'cache-only',
```

Comment:
This policy is required because the result of the query invalidates a part of the cache that causes another query to re-run, re-running this too